### ISSUE TITLE
使用自己的数据测试

### ISSUE THREAD

**Issue by WSTao** [liuruijin17/LSTR#8]
非常感谢作者的工作，我想使用自己的图片测试（大小和tusimple的不一样），但得到的结果很离谱。我应该修改那些参数呢

**Comment by liuruijin17** [comment]
You first need to crop your images to make sure the aspect ratio is 16 (width) to 9 (height). Then run the code again. 
Our codes will automatically interpolate your input image to 640x360, so the prerequisite is that the image before resizing is 16 to 9 like TuSimple (1280x720).

**Comment by WSTao** [comment]
好的，非常感谢！

**Comment by WSTao** [comment]
您好，我把自己的图片resize和tusimple大小一致，测试效果还是很一般，但tusimple的数据测试很正常，唯一的区别在于相机的安装角度，导致车道线消失点和tusimple的差别较大，导致曲线拟合效果变差。不知道您有没有测试过，算法的鲁棒性如何呢。算法在特殊的场景表现如何，比如上坡下坡以及弯道，谢谢！

**Comment by liuruijin17** [comment]
We have tested on lots of self-collected images and we also found:
1. regardless of the scenes, the more similar the lanes in the image are to TuSimple, the better the detection;
2. when the skyline moves up and down a lot, the results are not good;

In fact, these results were predictable. After all, TuSimple has limited data and cannot provide more information for model learning. For non-TuSimple data, it is better to have supervised learning on them.

As for special scenes, up and down hills and curves are also present in TuSimple, and the performance is good.
